akka-stream/src/main/scala/akka/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -368,8 +368,13 @@ private final case class SavedIslandData(islandGlobalOffset: Int, lastVisitedOff
 
   private[this] def createFlowName(): String = flowNames.next()
 
-  /** INTERNAL API */
-  private[akka] val defaultInitialAttributes = {
+  /**
+   * Default attributes for the materializer, are always seen as least specific, so any attribute
+   * specified in the graph or on the stages "wins" over these
+   *
```

Comment:
Rebasing this on top of the other one is a bit hellish... I'll leave it to you @johanandren -- don't want to pick the wrong wordings